access_point/server/README.md

```diff
@@ -16,14 +16,16 @@ This server can be access on port 5000 when on the same network and `http:10.0.0
  * 'wifissid' is the SSID of the WIFI network you want to configure
  * 'wifipassd' is the PASSWORD of that WIFI network
 
-* /config/'stt'/'tts'/'hotword'/'wake'/'auth'/'email'/'passwd'
+* /config/'stt'/'tts'/'hotword'/'wake'
  * 'stt' is the name of Speech To Text Service Provider
   - You can choose from 'google', 'ibm', 'bing', 'sphinx'
  * 'tts' is the name of Text To Speech Provider
```

Comment:
the best thing would be to have tildes ( ` ) instead of simple quotation marks.
it will look good ?

Reply:
@prateekiiest , that would raise a codacy issue

Reply:
@prateekiiest , that would treat them as code blocks and I am not sure that we should do that

Reply:
ok, got it @stealthanthrax :+1: